engine/runner/src/main/scala/org/enso/runner/ContextFactory.scala

```diff
@@ -147,18 +146,8 @@ class ContextFactory {
     * Creates a temporary polyglot engine for that and makes sure that it is closed.
     */
   private def engineHasJava(): Boolean = {
-    Using(
-      Engine
-        .newBuilder()
-        .allowExperimentalOptions(true)
-        .out(new ByteArrayOutputStream())
-        .err(new ByteArrayOutputStream())
-        .build()
-    ) { engine =>
-      engine.getLanguages.containsKey("java")
-    } match {
-      case Success(ret) => ret
-      case Failure(ex)  => throw new IllegalStateException("unreachable", ex)
-    }
+    val modules = ModuleLayer.boot().modules().stream()
```

Comment:
Nice. This should also be much faster than building an `Engine` that will be discarded right away.

Reply:
My secret hope is that also debugging will be faster. Starting one extra `Engine` might have be the cause why Truffle Debugging support was slightly confused.